assessment/models.py

```diff
@@ -487,6 +487,32 @@ def get_assessee_progress_on_event(self, assessee: Assessee):
         event_participation = self.get_assessment_event_participation_by_assessee(assessee)
         return event_participation.get_event_progress()
 
+    def set_name(self, name):
+        self.name = name
+        self.save()
+
+    def set_start_date(self, start_date_time):
+        self.start_date_time = start_date_time
+        self.save()
+
+    def set_test_flow(self, test_flow):
+        self.test_flow_used = test_flow
+        self.save()
+
+    def has_been_attempted(self):
+        event_participations: List[AssessmentEventParticipation] = self.assessmenteventparticipation_set.all()
+        return any([event_participation.has_attempted_test_flow() for event_participation in event_participations])
+
+    def start_time_has_passed(self):
+        return self.start_date_time <= datetime.datetime.now(tz=pytz.utc)
+
+    def is_deletable(self):
```

Comment:
Hello, Jo. If I may ask, is this function exposed to the frontend? As in is this information passed to the requests?

Reply:
Hello, Alex. Apparently it is not yet passed in the front end. At the moment, the function will only be invoked whenever an assessor or company attempt to delete an assessment event. If it is needed, I can include the results in the AssessmentEvent serializer such that it will be sent along with other assessment event meta data.

Reply:
Okay jo, sounds helpful! If possible, when the function is passed to the frontend, I can perform conditional actions based on it.